sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 12.2.1 (Unreleased)
 
+**Notes**
+- The `StorageUserAgentPolicy` is now replaced with the `UserAgentPolicy` from azure-core.
```

Comment:
We should update this note to state the custom user agents will now be added as prefix rather than appended.